### PR TITLE
docs: Remote debugging using VS Code requires exporting DISPLAY in launch.json

### PR DESCRIPTION
**Pull request by mcNets**
GitHub Issue (If applicable): closes #16146

https://github.com/unoplatform/uno/discussions/16146

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

There is no information about this topic.

## What is the new behavior?

Add more info about this topic.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/using-skia-gtk.md [hunk 1]
```diff
@@ -99,6 +99,25 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 </ItemGroup>
 ```
 
+### Remote debuging
```
Comment (Youssef1313) [inline doc/articles/features/using-skia-gtk.md:102]:
```suggestion
### Remote debugging
```

File: doc/articles/features/using-skia-gtk.md [hunk 2]
```diff
@@ -99,6 +99,25 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 </ItemGroup>
 ```
 
+### Remote debuging 
+
+VS Code can [connect to a remote computer](https://code.visualstudio.com/docs/remote/ssh) (using ssh) to develop and debug projects. Note that all the sources, build and debugging are done on the remote computer, where the Uno Platform extension is being executed.
```
Comment (Youssef1313) [inline doc/articles/features/using-skia-gtk.md:104]:
```suggestion
VS Code can [connect to a remote computer](https://code.visualstudio.com/docs/remote/ssh) (using SSH) to develop and debug projects. Note that all the sources, build and debugging are done on the remote computer, where the Uno Platform extension is being executed.
```

File: doc/articles/features/using-skia-gtk.md [hunk 3]
```diff
@@ -99,6 +99,25 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
...
+
+To debug the application remotely, it's necessary to modify the launch.json file and add the DISPLAY variable in the env section:
```
Comment (Youssef1313) [inline doc/articles/features/using-skia-gtk.md:106]:
```suggestion
To debug the application remotely, it's necessary to modify the `launch.json` file and add the `DISPLAY` variable in the `env` section:
```

**Comment by spouliot** [comment]
This is great but, for 5.2 onward, it should also be part of `using-skia-desktop.md` in the **X11 Specifics** section.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Comment by jeromelaban** [comment]
Thanks for your contribution!

**Pull request merged**